reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: single
+title:  "Pipeline Stages"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+This article describes the currently-supported stages that you can add to your
+Spinnaker pipelines.
```

Comment:
Not sure what @dorbin 's stance on this is, but given that this is intentionally a bare-bones first pass, is it worth calling that out here?

Reply:
Not sure what you mean, @lwander . Are you referring to the "currently-supported" part?

Reply:
I would have recommended calling out that this is just a first pass, and that we will be adding details on each stage shortly. Right now each description is a rewording of the stage title and makes the doc look a little out-of-place in a reference section.